$NetBSD: patch-dependency-check_Rakefile,v 1.2 2018/04/13 16:31:10 tsutsui Exp $

- ignore native-package-installer dependency checks
  (pkgsrc will handle dependencies internally)

--- dependency-check/Rakefile.orig	2018-04-10 16:26:48.000000000 +0000
+++ dependency-check/Rakefile
@@ -29,13 +29,13 @@ end
 namespace :dependency do
   desc "Check dependency"
   task :check do
-    unless PKGConfig.check_version?("clutter-gst-3.0")
-      unless NativePackageInstaller.install(:debian => "libclutter-gst-3.0-dev",
-                                            :redhat => "pkgconfig(clutter-gst-3.0)",
-                                            :homebrew => "clutter-gst",
-                                            :msys2 => "clutter-gst")
-        exit(false)
-      end
-    end
+#    unless PKGConfig.check_version?("clutter-gst-3.0")
+#      unless NativePackageInstaller.install(:debian => "libclutter-gst-3.0-dev",
+#                                            :redhat => "pkgconfig(clutter-gst-3.0)",
+#                                            :homebrew => "clutter-gst",
+#                                            :msys2 => "clutter-gst")
+#        exit(false)
+#      end
+#    end
   end
 end
